apps/ensnode/.env.local.example

```diff
@@ -52,3 +52,7 @@ ACTIVE_PLUGINS=eth,base,linea
 # Read more about ENSRainbow here:
 # https://ensrainbow.io
 ENSRAINBOW_URL=https://api.ensrainbow.io
+
+# The ENSNode public service URL
+# It is used to generate self-referential URLs in the API responses
+ENSNODE_PUBLIC_URL=http://localhost:42069
```

Comment:
We need this one to handle a redirect to ENSAdmin in a way the later would know how to link to ENSNode